### PR TITLE
Workflow pub sub

### PR DESCRIPTION
**Pull request by jesilz**

### CODE REVIEW

File: lib/buweb/concerns/work_flow.rb [hunk 1]
```diff
@@ -56,26 +57,44 @@ module WorkFlow
       state :archived
 
       event :submit_for_review do
+        after do
+          broadcast(:object_submitted_for_review, self.class, self.id)
```
Comment (jesilz) [inline lib/buweb/concerns/work_flow.rb]: @rift137 is there anyway to pass the current user if one is defined? How do classes with Mongoid::History::Trackable access `current_user` to set the `modifer_id` ?

Comment (rift137) [inline lib/buweb/concerns/work_flow.rb]: On update you should just be able to pass through `modifier: current_user`

Comment (jesilz) [inline lib/buweb/concerns/work_flow.rb]: oh, so any class with Mongoid::History::Trackable has a `modifier` field so I can just pass that through in the broadcast. I'm not concerned with setting it here, just passing it through.

File: buweb_content_models.gemspec [hunk 2]
```diff
@@ -25,9 +25,11 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'aasm', '~> 3.3'
   s.add_runtime_dependency 'mini_magick', '~> 3.7'
   s.add_runtime_dependency 'lisbn', '~> 0.2'
+  s.add_runtime_dependency 'wisper', '2.0.0.rc1'
```
Comment (halloffame) [inline buweb_content_models.gemspec]: I wonder we should allow for a range of versions here.
Maybe this should be `'>= 2.0.0.rc1', '< 2.1'`

**Pull request merged**